### DISCUSSION TITLE
Platform::WindowlessEglApplication::tryCreateContext(): no EGL devices found, likely a driver issue

### DISCUSSION THREAD

**Discussion by YuminosukeSato** [facebookresearch/habitat-lab#1325] [Q&A]
I have used WSL on Windows 10 to run Ubuntu 22.04 and started habitant-lab using Docker. I followed the steps correctly, but I received the following error when running "python examples/example.py":
"eglQueryDevicesEXT(): EGL_BAD_ALLOC error: In eglQueryDevicesEXT: Failed to allocate device list.
Platform::WindowlessEglApplication::tryCreateContext(): no EGL devices found, likely a driver issue
WindowlessContext: Unable to create windowless contexts"
Nvidia-smi is displayed, and I have executed the following commands:
"export MAGNUM_LOG=verbose
export MAGNUM_GPU_VALIDATION=ON"
Please provide a solution to this issue.

**Comment by edwardjjj** [comment]
I faced the same issue. After spending a week trying to fix the problem, I finally gave up and installed Ubuntu alongside Windows, and just ran simulation on Ubuntu.

**Comment by RayDebjit** [comment]
I am not expert about docker, but you should run on separate ubuntu system because there you can easily fix this issue by reinstalling drivers

**Comment by figkim** [comment]
If you’re using a Docker environment, please check your base image configuration. The commonly used nvidia-cuda images typically don’t support EGL. I resolved this issue by switching to the cudagl image (docker://nvcr.io#nvidia/cudagl). Give it a try!
